src/Server.jl

```diff
@@ -743,6 +743,8 @@ function make_pull_request(pp::ProcessedParams, rp::RequestParams, rbrn::RegBran
     catch ex
         if is_pr_exists_exception(ex)
             @debug("Pull request already exists, not creating")
+            params = Dict("body" => params["body"])
```

Comment:
You need more params here, in particular the PR number since the call on the next line wouldn't know what PR to update.

Reply:
Oh right, of course.

Reply:
I think this might do the right thing now